lib/dal/src/attribute/prototype/argument.rs

```diff
@@ -127,7 +123,6 @@ pub type AttributePrototypeArgumentResult<T> = Result<T, AttributePrototypeArgum
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 pub struct AttributePrototypeArgument {
     id: AttributePrototypeArgumentId,
-    targets: Option<ArgumentTargets>,
```

Comment:
This is the principal change: removed targets here, and all code that uses it (which is our socket connections code).